integration_tests/sdk/README.md

```diff
@@ -31,21 +31,23 @@ see `aqueduct_tests/conftest.py` and  `data_integration_tests/conftest.py`.
 
 ## Commands
 
-From this directory, to run the Aqueduct Tests:
-`pytest aqueduct_tests/ -rP -vv`
+To run all SDK Integration Tests:
```

Comment:
Does it matter which directory the the script is run from? I think it does based on the arg parsing below? If so, can you add instructions to this.